### PR TITLE
tasks-administer-federation-deployment-2017-12-18

### PR DESCRIPTION
**Pull request by lichuqiang**

### CODE REVIEW

File: docs/tasks/administer-federation/deployment.md [hunk 1]
```diff
@@ -1,76 +1,133 @@
...
+[Kubernetes Deployment](/docs/concepts/workloads/controllers/deployment/) 非常类似，
+并提供相同的功能。 在联邦控制平面中创建联邦 Deployment 确保所需的副本数存在于注册的群集中。
 
+<!--
 **As of Kubernetes version 1.5, Federated Deployment is an Alpha feature. The core
 functionality of Deployment is present, but some features
 (such as full rollout compatibility) are still in development.**
+-->
+**到 Kubernetes 1.5 版本为止，联邦 Deployment 还是一个 Alpha 特性。 Deployment 的核心功能已经提供，
+但一些特性（例如完整的部署兼容性）仍在开发中。**
```
Comment (zhangxiaoyu-zidif) [inline docs/tasks/administer-federation/deployment.md]: rollout是deployment的一个功能.这里翻译不妥。

**Comment by zhangxiaoyu-zidif** [comment]
@tengqm The translation work has been changed here. If interested, you can review here.

File: docs/tasks/administer-federation/deployment.md [hunk 2]
```diff
@@ -1,76 +1,133 @@
...
 {% capture prerequisites %}
 
-* {% include federated-task-tutorial-prereqs.md %}
+* {% include federated-task-tutorial-prereqs.md %}
+<!--
 * You should also have a basic
 [working knowledge of Kubernetes](/docs/setup/pick-right-solution/) in
 general and [Deployments](/docs/concepts/workloads/controllers/deployment/) in particular.
+-->
+* 通常您还应当拥有基本的 [ Kubernetes 应用知识](/docs/setup/pick-right-solution/)，特别是 [Deployment](/docs/concepts/workloads/controllers/deployment/) 相关的应用知识。
```
Comment (zhangxiaoyu-zidif) [inline docs/tasks/administer-federation/deployment.md]: [ Kubernetes 应用知识]多了一个空格。

**Comment by lichuqiang** [comment]
@zhangxiaoyu-zidif updated, thanks.

**Pull request merged**